packages/nimble-components/src/wafer-map/tests/wafer-map.spec.ts

```diff
@@ -100,4 +100,57 @@ describe('WaferMap', () => {
         DOM.processUpdates();
         expect(spy).toHaveBeenCalledTimes(1);
     });
+
```

Comment:
Could you do a bit of refactoring to reduce duplication? Some ideas:
1. a helper function for `element.zoomContainer.getAttribute('transform')?.toString();`
2. move the duplicated lines at the beginning of each test into a `beforeEach()` function (would require a new `describe()` block for these 3 tests